.github/CHANGELOG.md

```diff
@@ -30,7 +33,7 @@
 
 ### Contributors
 
-Amintor Dusko, Lee James O'Riordan, Chae-Yeun Park
+Amintor Dusko, Lee James O'Riordan, Chae-Yeun Park, Vincent Michaud-Rioux
```

Comment:
The convention for names order is last name in alphabetical order.
```suggestion
Amintor Dusko, Vincent Michaud-Rioux, Lee James O'Riordan, Chae-Yeun Park
```

Reply:
Duly noted, thanks @AmintorDusko .